TimeLog.API.Documentation/wwwroot/Reporting/GetWorkUnitsRaw.xsd

```diff
@@ -48,7 +49,6 @@
       <xsd:element name="EstimatedAmount" type="xsd:decimal" />
       <xsd:element name="EstimatedAmountProjectCurrency" type="xsd:decimal" />
       <xsd:element name="BARHours" type="xsd:decimal" /><!-- BAR: Booked as revenue -->
-      <xsd:element name="BARHoursProjectCurrency" type="xsd:decimal" />
```

Comment:
Do we still need to have GetWorkUnitRaw since we already replaced it with GetWorkUnitRawPaged in the background process? just a thought

Reply:
Initially we attempted to replace the GetWorkUnitRaw method by adding pagination, but Troels pointed out that customers using BI tools access this method via HTTP GET. When using HTTP GET, all required fields must be present, or the request will fail. 
To avoid introducing breaking changes, we decided to introduce a new method rather than modifying the existing one. This ensures that customers still using GetWorkUnitRaw remain unaffected. The new GetWorkUnitRawPaged method will also be beneficial for handling large payloads, as it allows the data to be split across multiple pages.